docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial presents the basic flow in which you manually register an external Application's API into Compass and expose it into the Kyma Runtime. In the Kyma Runtime, you then create a lambda that calls the `order-service` API. While going through this tutorial, you will navigate between two UI views:
+
+- Compass UI, where you create connections between Applications, Runtimes, and Scenarios
+- Kyma Console UI, where you manage resources used in your Application, such as services, lambdas, and bindings
+
+## Prerequisites
+
+For simplicity reason, use the available Order Service as the sample external Application for this tutorial. Prepare the following:
+
+- [`order-service`](./assets/order-service.yaml) file that contains the service definition, deployment, and its API
+- [API specification](./assets/order-service-api-spec.yaml) of the `order-service`
+- [Lambda function](./assets/lambda.yaml) that calls `order-service` for orders
+- Kyma cluster with the Compass module enabled
+
+>**NOTE:** Read [this](#installation-enable-compass-in-kyma-default-kyma-installation) document to learn how to install Kyma with the Compass module.
+
+## Steps
+
+### Deploy the external Application
+
+1. Log in to the Kyma Console and create a new Namespace by selecting the **Add new namespace** button.
+
+2. In the **Overview** tab in your Namespace, select the **Deploy new resource** button and use the [`order-service`](./assets/order-service.yaml) file to connect the Application.
+
+3. In the **APIs** tab, copy the URL to your Application. You will use it in the Compass UI in the next steps.
+
+### Register your Application in the Compass UI
+
+1. Select **Back to Namespaces** in the top-left corner of the page and go to the **Compass** tab in the left navigation panel. It will navigate you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial.
```

Comment:
```suggestion
1. Select **Back to Namespaces** in the top-left corner of the page, and go to the **Compass** tab in the left navigation panel. It will navigate you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial.
```